classes/person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parents_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parents_permission = parents_permission
+  end
+
+  def of_age?(age)
+    @age = age
+    @age >= 18
+  end
+
+  private :of_age?
```

Comment:
- [ ] You did a great job, Classes are working properly! 🎉 However, let's make it even better by using the Ruby best practices for syntax, usually when your Class has a private method you place it at the bottom like this:

```
class Person

  def initialize
      ...
  end

  def can_use_service? 
     ...
  end

  private 

  def of_age?
    ...
  end
end
```
With this syntax, you can change `private :of_age?` to `private` and then place the private methods below that line.

Reply:
@smunozmo Thank you for the review I will update the code now.